src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -162,7 +162,13 @@ public static void init(RobotID id) {
 			RIGHT_FOLLOWER_2_CHANNEL = 6;
 
 			HAS_ELEVATOR = true;
-			ELEVATOR_MOTOR_CHANNEL = 0;
+			ELEVATOR_MOTOR_CHANNEL = 7;
+
+			ELEVATOR_BOTTOM_TICKS = 764;
```

Comment:
Add TODO comment saying that we need to gather the actual correct elevator tick readings for this robot.